README.md

```diff
@@ -186,6 +184,17 @@ The following commands help you understand and effectively use the Tekton CLI:
 
 For every `tkn` command, you can use `-h` or `--help` flags to display specific help for that command.
 
+## Disable Color and Emojis in Output 
+
+For many `tkn` commands, color and emojis by default will appear in command output. To remove the color and emojis 
+from all `tkn` command output, set the environment variable `NO_COLOR`, such as shown below:
```

Comment:
maybe we want to mention the --no-color command line option too ? for "ad-hoc" disablement of the colouring ?

Reply:
Yup, makes sense to also document `--nocolour`. Agree, a bit confusing with the two different approaches and maybe we should reconsider something around #992. 

Pushed up the additional documentation.

Reply:
Having at least an alias to both ? and only document the american webster way ?

Reply:
Let's raise an issue for this to dicuss it there.

Reply:
Yeah, that's what I was thinking. Document it as is for now and adjust later on once there is a clear strategy for it. But I agree that an alias approach makes sense.